content/integrations/sso/sso-jumpcloud.md

```diff
@@ -8,10 +8,10 @@ weight: 2
 ---
 
 {{% alert success %}}
-Single Sign-On (SSO) is a **paid-only feature**. Wish to upgrade? Contact your [Sales Representative](mailto:sales@metricly.com).
+Single Sign-On (SSO) is a **paid-only feature**. Wish to upgrade? Contact your [Sales Representative](mailto:cloudwisdom.sales@virtana.com).
```

Comment:
Same here.

Reply:
for this one, I think bob is having that alias created today.

Reply:
Yup, just got to confirm that it gets made. Should be soon.